tests/test-http-stream-finish.lua

```diff
@@ -1,7 +1,16 @@
 local http = require("http")
 
 require('tap')(function(test)
-  test('http stream end', function(expect)
+
```

Comment:
I can do this, but i will add this test again later. I will do what if you
want, just confirm this again :)

2015-04-20 23:03 GMT-03:00 Ryan Phillips <notifications@github.com>:

> In tests/test-http-stream-finish.lua
> <https://github.com/luvit/luvit/pull/702#discussion_r28745203>:
>
> > @@ -1,7 +1,16 @@
> >  local http = require("http")
> >
> >  require('tap')(function(test)
> > -  test('http stream end', function(expect)
> > +
>
> removing it seems fine... and renaming 'http stream end #2
> <https://github.com/luvit/luvit/issues/2>' back to #1
> <https://github.com/luvit/luvit/issues/1>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/luvit/luvit/pull/702/files#r28745203>.
>